### PR TITLE
WEBUI-879: select all remains active when unselecting individual docu…

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
https://jira.nuxeo.com/browse/WEBUI-879

### CODE REVIEW

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good

File: elements/nuxeo-results/nuxeo-results.js [hunk 1]
```diff
@@ -745,6 +745,13 @@ Polymer({
     }
   },
 
+  _isChecked(selectAllActive, _excludedDocs) {
+    if (selectAllActive && _excludedDocs === 0) {
```
Comment (richardsd) [inline elements/nuxeo-results/nuxeo-results.js:749]: You could probably simplify the code with:

```js
_isChecked(selectAllActive, _excludedDocs) {
  return selectAllActive && _excludedDocs === 0;
}
```

wdyt?

**Review by richardsd — APPROVED** [review]
It looks good. Just left you a comment to simplify the code 👍 Nice work

File: elements/nuxeo-results/nuxeo-results.js (hunk 1 above)
Comment (rakeshkumar1019) [reply elements/nuxeo-results/nuxeo-results.js:749]: thankyou @richardsd

**Review by rajatgoel-gl — APPROVED** [review]
Do implement the changes stated by Richard. Rest looks good

**Pull request merged**